lib/y2_installbase.pm

```diff
@@ -62,6 +63,30 @@ sub accept_changes {
     assert_screen 'inst-overview';
 }
 
+=head2 validate_default_target
+
+    validate_default_target();
```

Comment:
Fixed missing parameter. As this function is added in y2_installbase, I guess it is clear that it can be used for installation only. As it selects `install-shell`, it cannot be called on installed system. 

In general, I assume it as temporary solution and once we make all the test suites to use the modules with libyui, we can remove `installation_overview` and all the internals of this function may be moved to `validate_default_target.pm` test module.